docs/L1_L2_Messages.md

```diff
@@ -0,0 +1,233 @@
+---
+id: L1_L2_Messages
+title: Messaging Between Layers
+sidebar_label: qqq
+---
+
+The Arbitrum protocol offers trustless message passing between the layer 1 and layer 2 chains. The most common use-case for inter-chain communication is depositing and withdrawing assets; this, however, is only one specific application of generalized cross-chain contract calls that Arbitrum supports. This page covers the generalized protocol; for further explanation, see [Inside Arbitrum: Bridging](https://developer.offchainlabs.com/docs/inside_arbitrum#bridging).
+
+## Ethereum to Arbitrum: Retryable Tickets
+
+#### Explanation
+
+Arbitrum offers several ways for an Ethereum transaction to send a message to Arbitrum ([see "L2 Messages"](Data_Formats.md)); however, the generally recommended method to use for direct L1 to L2 communication is via retryable tickets.
+
+The idea is the following: a layer 1 transaction includes ABI encoded data of a message to be executed on Arbitrum, along with some Ether to be used as ArbGas for its execution. The message gets inserted into the Arbitrum chain's inbox, where it will be executed like any other L2 message. In the uncommon / less happy case in which it reverts, the transaction gets put into an L2 "retry buffer." This means that for a period of time (likely on the scale of the chain's dispute window, so roughly one week), anybody can attempt to "redeem" the the L2 transaction ticket by re-executing it.
```

Comment:
It seems to me more natural to say that do an L1 tx to the inbox, to submit an L2 transaction (including calldata, callvalue, and gas info) in such a way that if the L2 tx can't execute successfully, a retryable ticket is created at L2, etc.